keyboards/hidtech/bastyl/jsons/bastyl.json

```diff
@@ -0,0 +1,315 @@
+{
```

Comment:
This file belongs in the VIA repo, not here:

https://github.com/the-via/keyboards